include/disccord/rest/api_client.hpp

```diff
@@ -160,7 +161,7 @@ namespace disccord
 
                     pplx::task<void> modify_guild_member(uint64_t guild_id, uint64_t user_id, disccord::rest::models::modify_guild_member_args args, const pplx::cancellation_token& token = pplx::cancellation_token::none());
 
-                    //TODO: modify_current_nick
+                    pplx::task<web::json::value> modify_current_nick(uint64_t guild_id, std::string nick, const pplx::cancellation_token& token = pplx::cancellation_token::none());
```

Comment:
According to discord, this returns a json response with the nickname on success, so you could probably return a model for this